app/client/src/features/events/EventLive.tsx

```diff
@@ -18,11 +18,13 @@ import { SpeakerList } from './Speakers';
 const useStyles = makeStyles((theme) => ({
     root: {
         height: '100%',
+        padding: theme.spacing(0, 3, 0, 3), // add side padding since layout padding is disabled
         [theme.breakpoints.up('md')]: {
             flexDirection: 'row',
             flexWrap: 'nowrap',
         },
         [theme.breakpoints.down('sm')]: {
+            padding: theme.spacing(3, 3, 0, 3), // add top padding so event video doesn't touch navbar
```

Comment:
I think ideally, these padding changes should exist in `pages/events/[id]/live`.  It also makes more sense that the file where you're passing in the `disablePadding` also explicitly defines what the new padding is.